department-of-veterans-affairs/next-build
# Fix: Issue 663 social links

# Description
addresses feedback from the UX design team. We wanted to make sure we sent the right link to social media when shared out. 

## Ticket
Relates to [Issue 663](https://github.com/department-of-veterans-affairs/next-build/issues/663)

## Developer Task

```[tasklist]
```

## Testing Steps
Locally:

1. start yarn dev
2. from the home page go to the footer
3. click outreach events
4. click an event
5. use the element inspector on the "add to google", "share on facebook", and "share on twitter" links
6. verify that the href contains the link given the current testing environment.

In DEV
I published the code for this change to the development environment to test it. The result was that the links work as expected.
![image](https://github.com/user-attachments/assets/3f3c8989-9ad3-4e7a-94aa-ab2e93453141)



## Screenshots

Before:
![image](https://github.com/user-attachments/assets/fe28b5eb-23c2-43eb-9ba3-7c1ad7a259d4)


After:
![image](https://github.com/user-attachments/assets/d5ed6da4-3207-4992-88ed-1eac389891b2)
# Reviewer

### Reviewing a PR

This section lists items that need to be checked or updated when making changes to this repository. 

## Standard Checks

```[tasklist]
```

## Merging an Approved Layout

When merging a layout, you must ensure that the content type has been turned on for `next-build` inside the `CMS`. This CMS flag must be turned on for editors to preview their work using the next build preview server.

Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved.

The status of layouts should be kept up to date inside [templates.md](./templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.
